### DISCUSSION TITLE
How to get a regulator-fixed initializing in off-state

### DISCUSSION THREAD

**Discussion by jbr-ia** [zephyrproject-rtos/zephyr#77542] [Q&A]
I see the regulator-fixed can be set to initialize to on in the device-tree by adding: regulator-boot-on;

Is there also a possibility to force the regulator to start off?
If the IO-pins are initialized with ACTIVE_HIGH it works fine, but I have a regulator on a custom board with stm32wb55rg microcontroller which has the IO-pins as ACTIVE_LOW. The regulator is described in the device-tree as:

	i2c_pwr: i2c-pwr-ctrl {
		compatible = "regulator-fixed";
		regulator-name = "i2c-pwr-ctrl";
		enable-gpios = <&gpiob 5 GPIO_ACTIVE_LOW>;
		off-on-delay-us = <100000>;
	};
	
After initialisation, the physical pin is low (and therefore the physical regulator is active active). But the regulator-fixed in zephyr  thinks it is disabled. In order to really disable it, I now enable and disable the regulator in the code. But I think I am missing some option in the device-tree to let the initialisation disable the regulator. I am using zephyr v3.7.0

**Comment by rcolatobe** [comment]
@jbr-ia For some reason it didn't link to this post, I have #89973 open that addresses this issue.
